resources/phing/AtoumTask.php

```diff
@@ -180,7 +180,7 @@ public function execute()
 
 		if ($this->telemetryEnabled())
 		{
-			if (class_exists('reports\telemetry') === false) {
+			if (class_exists('mageekguy\atoum\reports\telemetry') === false) {
```

Comment:
Is it possible to have any side effect with "magical" `mageekguy\atoum` to `atoum` translation ?